### PR TITLE
Tickets/sitcom 807

### PR DESCRIPTION
**Pull request by edennihy**

### CODE REVIEW

File: bash/lsst/sitcom/user_setups_mod.sh
Comment (tribeiro) [inline bash/lsst/sitcom/user_setups_mod.sh:1]: Do we really need to keep all this old code commented out? git will remember that hehehe

File: bash/lsst/sitcom/user_setups_mod.sh [hunk 1]
```diff
@@ -2,79 +2,84 @@
...
+# where file with setup statements are held
+REPOS_FILE_PATH=$REPOS"auto_env_setup.sh"
+
+cat <<EOT >> $FILE_PATH
+#!/usr/bin/sh bash
+# This file is auto generated by the ${HOME}/auxtel_obsruns_env_setups/bash/lsst/sitcom/user_setups_mod script. The first three lines are added automatically and source the shared common environment needed to use Notebooks during an observing run. DO NOT overwrite, modify, or move. 
+source ${REPOS_FILE_PATH}
+
+# The following three lines are required to use a DDS daemon within a notebook.
+export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
```
Comment (tribeiro) [inline bash/lsst/sitcom/user_setups_mod.sh]: This is no longer needed. The value that ships with the container is ok now.

**Review by tribeiro — APPROVED** [review]
Look good! But I left two comments for you. I hope you will consider removing all the commented code, git remembers. 

The second comment is more critical. We no longer need the `export OSPL_HOME` as the value provided by the container is correct now.

File: bash/lsst/sitcom/user_setups_mod.sh
Comment (edennihy) [reply bash/lsst/sitcom/user_setups_mod.sh:1]: Unused code removed.

**Pull request merged**